applications/serial_lte_modem/src/nrfcloud/slm_at_nrfcloud.c

```diff
@@ -24,31 +25,28 @@ LOG_MODULE_REGISTER(slm_nrfcloud, CONFIG_SLM_LOG_LEVEL);
 #define MODEM_AT_RSP \
 	"{\"appId\":\"MODEM\", \"messageType\":\"RSP\", \"data\":\"%s\"}"
 
-/**@brief nRF Cloud operations. */
-enum slm_nrfcloud_operation {
-	SLM_NRF_CLOUD_DISCONNECT,
-	SLM_NRF_CLOUD_CONNECT,
-	SLM_NRF_CLOUD_SEND,
-	SLM_NRF_CLOUD_CELLPOS_STOP         = SLM_NRF_CLOUD_DISCONNECT,
-	SLM_NRF_CLOUD_CELLPOS_START_SCELL  = SLM_NRF_CLOUD_CONNECT,
-	SLM_NRF_CLOUD_CELLPOS_START_MCELL  = SLM_NRF_CLOUD_SEND,
-	SLM_NRF_CLOUD_WIFIPOS_STOP         = SLM_NRF_CLOUD_DISCONNECT,
-	SLM_NRF_CLOUD_WIFIPOS_START        = SLM_NRF_CLOUD_CONNECT
-};
-
 static struct k_work cloud_cmd;
 static K_SEM_DEFINE(sem_date_time, 0, 1);
 
 #if defined(CONFIG_NRF_CLOUD_LOCATION)
-static struct k_work cell_pos_req;
-static struct k_work wifi_pos_req;
-static enum nrf_cloud_location_type cell_pos_type;
-static uint64_t ttft_start;
-static enum {
-	RUN_TYPE_NONE,
-	RUN_TYPE_CELL_POS,
-	RUN_TYPE_WIFI_POS
-} run_type;
+
+/* Cellular positioning services .*/
+enum slm_nrfcloud_cellpos {
+	CELLPOS_NONE, /* Keep first (== 0). */
+	CELLPOS_SINGLE_CELL,
+	CELLPOS_MULTI_CELL,
+
+	CELLPOS_COUNT /* Keep last. */
+};
+/* Whether cellular positioning is requested and if so, which. */
+static enum slm_nrfcloud_cellpos s_cell_pos = CELLPOS_NONE;
```

Comment:
Prefixing static variables with s_ does not seem to be well established in applications or samples.